it/src/main/java/org/corfudb/universe/universe/vm/VmUniverse.java

```diff
@@ -56,17 +59,14 @@ public VmUniverse deploy() {
      */
     @Override
     protected Group buildGroup(GroupParams groupParams) {
-        switch (groupParams.getNodeType()) {
-            case CORFU_SERVER:
-                return VmCorfuCluster.builder()
-                        .universeParams(universeParams)
-                        .params(ClassUtils.cast(groupParams))
-                        .vms(applianceManager.getVms())
-                        .build();
-            case CORFU_CLIENT:
-                throw new UniverseException("Not implemented corfu client. Group config: " + groupParams);
-            default:
-                throw new UniverseException("Unknown node type");
+        if (groupParams instanceof CorfuServerParams) {
```

Comment:
try to avoid using instanceof (enum way better in this case)

Reply:
Fixed.